solutions/cf-1702E.mdx

```diff
@@ -12,7 +12,7 @@ author: Aadit Ambadkar
 **Time Complexity:** $\mathcal{O}(n)$
 
 First observe that if any number is present on more than 2 dominoes or if a dominoe has the same number repeated twice, then 
-it is impossible to create the 2 sets.
+it is impossible to create the 2 sets. _The proof of this is left as an exercise to the reader._
```

Comment:
...aadit i meant that as
"proofs left as exercise to the reader are awful and should never be used"

Reply:
if you read the problem is incredibly obvious why. Your asking me to essentially prove that you cannot fit 3 items in 2 groups, with each item going into a different group. Please consider that.

Reply:
😮‍💨 fine